tests/baselines/reference/user/zone.js.log

```diff
@@ -0,0 +1,7 @@
+Exit Code: 1
+Standard output:
+node_modules/zone.js/dist/zone.js.d.ts(292,36): error TS2304: Cannot find name '_PatchFn'.
```

Comment:
OK, looks expected and should be fixed in their next release by https://github.com/angular/angular/pull/31866